chapters/ch05.md

```diff
@@ -1,78 +1,99 @@
 
 ->
+Bạn nhận ra rằng các bức ảnh được người dùng tải lên nhìn khác các bức ảnh mà bạn tải về từ các trang web cho tập dữ liệu của bạn: đó là vì các bức ảnh được chụp bằng điện thoại thường sẽ có độ phân giải thấp hơn, bị nhòe (mờ) hoặc bị tối hơn. Do bộ nhận dạng được huấn luyện trên tập dữ liệu từ ảnh trên các trang web nên nó không khái quát hóa tốt đến phân phối thực tế mà bạn cần nhắm đến, đó là ảnh chụp từ điện thoại.
 
 
 Before the modern era of big data, it was a common rule in machine learning to use a random 70%/30% split to form your training and test sets. This practice can work, but it’s a bad idea in more and more applications where the training distribution (website images in our example above) is different from the distribution you ultimately care about (mobile phone images).
 
 ->
+Trước kỷ nguyên của big data, ta thường áp dụng việc chia ngẫu nhiên 70%/30% tập dữ liệu cho tập huấn luyện và kiểm tra trong machine learning. Tuy nhiên, đây không phải là một giải pháp phù hợp cho các ứng dụng mà dữ liệu huấn luyện (ảnh từ các trang web ở ví dụ trước) khác biệt về phân phối so với dữ liệu thực tế (ảnh chụp từ điện thoại).
 
 We usually define:
 
 ->
+Ta thường định nghĩa như sau:
 
 * **Training set​** — Which you run your learning algorithm on.
 
 ->
+* **Tập huấn luyện​** — Là tập dữ liệu để chạy thuật toán học.
 
 * **Dev (development) set​** — Which you use to tune parameters, select features, and make other decisions regarding the learning algorithm. Sometimes also called the hold-out cross validation set​.
 
 ->
+* **Tập phát triển​** — Là tập dữ liệu được dùng để điều chỉnh lại các tham số, lựa chọn đặc trưng và quyết định các thay đổi đến thuật toán học. Đôi khi, nó còn được gọi là tập kiểm định chéo​.
 
 * **Test set​** — which you use to evaluate the performance of the algorithm, but not to make any decisions regarding what learning algorithm or parameters to use.
 
 ->
+* **Tập kiểm tra​** — Là tập dữ liệu dùng để đánh giá chất lượng của thuật toán học, nhưng không được dùng để quyết định các thay đổi đến thuật toán học hay các tham số.
 
 Once you define a dev set (development set) and test set, your team will try a lot of ideas, such as different learning algorithm parameters, to see what works best. The dev and test sets allow your team to quickly see how well your algorithm is doing.
 
 ->
+Sau khi chia tập phát triển và tập kiểm tra, nhóm của bạn có thể thử nhiều ý tưởng khác nhau, ví dụ như các tham số khác nhau cho thuật toán học, để tìm ra cái tốt nhất. Tập phát triển và tập kiểm tra cho phép nhóm của bạn có thể đánh giá khả năng hoạt động của thuật toán một cách nhanh chóng.
 
 In other words, ​**the purpose of the dev and test sets are to direct your team toward the most important changes to make to the machine learning system​**.
 
 ->
+Nói cách khác, ​**mục đích của tập phát triển và tập kiểm tra là chỉ ra những sự thay đổi quan trọng nhất để tạo ra hệ thống machine learning​**.
```

Comment:
Chỗ "important changes to make to the machine learning system" ý là "make the important changes to the machine learning system" chứ không phải là "to make the machine learning systems". Chú ý từ "to" sau từ "make".

Đề xuất dịch:
"...tập kiểm tra là định hướng nhóm của bạn tới những thay đổi quan trọng nhất trong hệ thống machine learning."